.github/workflows/check-catalyst.yaml

```diff
@@ -31,7 +31,6 @@ jobs:
         run: |
```

Comment:
Let's also add a TODO for the future reference then.

Reply:
```suggestion
        run: |
        # TODO: Temporarily disable lightning.kokkos build due to https://github.com/actions/runner-images/issues/8659
```

Reply:
```suggestion
        run: |
          # TODO: reenable kokkos builds
```